Tests/SwiftParserTest/translated/SwitchTests.swift

```diff
@@ -35,7 +35,13 @@ final class SwitchTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(locationMarker: "1️⃣", message: "expected expression and '{}' to end 'switch' statement", fixIts: ["insert expression and '{}'"]),
         DiagnosticSpec(locationMarker: "2️⃣", message: "expected identifier and function signature in function", fixIts: ["insert identifier and function signature"]),
-      ]
+      ],
+      fixedSource: """
+        func parseError1(x: Int) {
+          switch <#expression#> {
+        }func <#identifier#>() {}
```

Comment:
We should be inserting a newline after the `}` to separate it from `func`.

Reply:
https://github.com/apple/swift-syntax/issues/1612